services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/sidecar.py

```diff
@@ -373,6 +379,23 @@ def get_dynamic_sidecar_spec(
                 f"node.labels.{DOCKER_TASK_EC2_INSTANCE_TYPE_PLACEMENT_CONSTRAINT_KEY}=={ec2_instance_type}",
             )
         )
+    if placement_settings.use_generic_resources_instead_of_placement_constraints:
+        replacement_constraints: dict[
+            str, PlacementConstraintStr
+        ] | None = (
+            placement_settings.DIRECTOR_V2_PLACEMENT_CONSTRAINTS_REPLACEMENTS_FOR_GENERIC_RESOURCES
+        )
+        if replacement_constraints:
+            for resource_name in scheduler_data.service_resources:
+                if resource_name not in {"CPU", "RAM"}:
+                    if resource_name not in replacement_constraints:
+                        msg = (
+                            "Since replacement placement constraints are enabled, "
```

Comment:
See above, with the wording.

I dont think it is clear what "replacement placement constraints" means except the author :D

Reply:
used some better naming